phoenix-core/src/main/java/org/apache/phoenix/util/PhoenixRuntime.java

```diff
@@ -632,6 +645,16 @@ public static ExecutionCommand parseArgs(String[] args) {
             return execCmd;
         }
 
+        private static String validateTableName(String tableName) {
```

Comment:
Looking at this method it seems like table names can't have : in them. Does the change in PTableKey.java still make sense then?